source/CMakeLists.txt

```diff
@@ -103,19 +101,13 @@ set(SOURCE_FILES
   FrameBase.cpp
   CmdLine.cpp
 
-  Configuration/PropertySheetHelper.cpp
-  Configuration/Config.cpp
-
-  linux/resources.cpp
-  linux/benchmark.cpp
+  linux/context.cpp
+  linux/linuxframe.cpp
   linux/paddle.cpp
-  linux/version.cpp
   linux/registryclass.cpp
-  linux/linuxframe.cpp
-  linux/context.cpp
-  linux/cassettetape.cpp
-  linux/network/slirp2.cpp
-  linux/network/portfwds.cpp
+  linux/resources.cpp
+  linux/soundbuffer.cpp
+  linux/version.cpp
```

Comment:
This directory should be moved. They're no longer linux-specific. I couldn't think of a good name. Maybe they should be moved into the `common2` folder?